forem/forem
# Archived post filters Migration

## What type of PR is this? (check all applicable)


## Description

In an effort to migrate away from asset pipeline, the current initializers need to be migrated to pack tags via Webpacker and initialized in the perspective view/page.

The initializeArchivedPostFilter() should be migrated to a pack tag and added to the perspective views in which this code should be initialized.

## Related Tickets & Documents

- Related Issue #https://github.com/forem/forem/issues/19242
- Closes #https://github.com/forem/forem/issues/19242

## QA Instructions, Screenshots, Recordings

Go to http://localhost:3000/dashboard and click on "Show archived" and "Hide Archived" and ensure you get the correct results. 

https://www.loom.com/share/3245351cabd5443c9df75c56f9abef85

### UI accessibility concerns?
N/A

## Added/updated tests?

- [x] No, I explain in the files view

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
